my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.post-title {
+  color: #37474f;
+  font-size: 16px;
+  line-height: 24px;
```

Comment:
Defining the line-height with a specific unit can lead bugs down the road. It's usually better to omit the unit, i.e. `line-height: 1.5`.

Reply:
Agreed. I see that now and I'll refactor to use the font-size as the basis for the line-height instead of using a unit.